state/txindex/kv/kv.go

```diff
@@ -61,6 +61,15 @@ func WithCompaction(compact bool, compactionInterval int64) IndexerOption {
 	}
 }
 
+// Close closes the indexer's underlying database. The caller is responsible for
+// calling Close when done with the indexer.
+func (txi *TxIndex) Close() error {
+	if err := txi.store.Close(); err != nil {
```

Comment:
This is closing the prefixDB? Anything special we need to do here?

Reply:
No, this closes the actual database. I did not add anything to close the `PrefixDB` because 1) it's not a "real" database, so nothing to close; 2) che changed the `Close` method of `PrefixDB` to be a no-op; 3) by closing the database, we also make `PrefixDB` unusable (as expected).

Reply:
Ok yes, store is the actual DB not the wrapper. GOod.